### PR TITLE
Add Overlays for tag description [skip-ci]

### PR DESCRIPTION
**Pull request by Guaris**
I added support for overlays for the tag descriptions since those will not be generated through any form of automation. 

https://konghq.atlassian.net/jira/software/c/projects/DOCU/boards/58?selectedIssue=DOCU-3499

### CODE REVIEW

File: api-specs/overlays/docs/docs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Installation
+
+We are using this [project](https://github.com/lornajane/openapi-overlays-js) for generation:
+
+* Clone [this](https://github.com/lornajane/openapi-overlays-js) repo, `cd` into the directory where you cloned it. 
+* Run `npm install` to get the dependencies
+* Run `npm install -g` to get `overlayjs` as a command
+
```
Comment (lena-larionova) [inline api-specs/overlays/docs/docs.md:13]:
```suggestion

1. Clone [this](https://github.com/lornajane/openapi-overlays-js) repo, `cd` into the directory where you cloned it. 
2. Run `npm install` to get the dependencies
3. Run `npm install -g` to get `overlayjs` as a command

```

File: api-specs/overlays/docs/docs.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Apply a transformation
+
+To apply an overlay to a spec use the following command:
```
Comment (lena-larionova) [inline api-specs/overlays/docs/docs.md:17]:
```suggestion
To apply an overlay to a spec, run the following command from your clone of the docs directory: 
```
Or something like "cd into your clone of the docs directory, then run the command:"

File: api-specs/overlays/docs/docs.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* Clone [this](https://github.com/lornajane/openapi-overlays-js) repo, `cd` into the directory where you cloned it. 
+* Run `npm install` to get the dependencies
+* Run `npm install -g` to get `overlayjs` as a command
+
+
+## Apply a transformation
+
+To apply an overlay to a spec use the following command: 
+
+`overlayjs --openapi openapi.yml --overlay overlay.yaml`
```
Comment (lena-larionova) [inline api-specs/overlays/docs/docs.md:19]: What's the purpose of showing how to output to the console? Won't we always be piping this output to a file?

File: api-specs/overlays/oss/api-description.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+actions:
+- target: $.info
+  update:
+    description: |-
+      <br>
+      <a href="https://insomnia.rest/run?label=Kong%20Gateway%20Open%20Source%203.4&uri=https%3A%2F%2Fraw.githubusercontent.com%2FKong%2Fdocs.konghq.com%2Fmain%2Fapi-specs%2FGateway-OSS%2F3.4%2Fkong-oss-3.4.json" target="_blank">
+      <img src="https://insomnia.rest/images/run.svg" alt="Run in Insomnia"></a> 
+      <br><br>
+
+      OpenAPI 3.0 spec for Kong Gateway's Enterprise Admin API.
```
Comment (lena-larionova) [inline api-specs/overlays/oss/api-description.yaml:14]:
```suggestion
      OpenAPI 3.0 spec for Kong Gateway's Open Source Admin API.
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

**Review by lena-larionova — APPROVED** [review]

**Comment by fabianrbz** [comment]
@Guaris let me know when you are planning to merge this, we need to prevent Jekyll from generating pages for the overlays

**Pull request merged**